### PR TITLE
Upgrade jiathis share thirdpard plugin code template.

### PR DESCRIPTION
**Pull request by elkan1788**
Upgrade jiathis share thirdpart plugin code template.

### CODE REVIEW

File: layout/_partials/share/jiathis.swig [hunk 1]
```diff
@@ -1,21 +1,25 @@
...
-  <a class="jiathis_counter_style"></a>
+<span class="jiathis_txt">分享到：</span>
+<a class="jiathis_button_fav">收藏夹</a>
+<a class="jiathis_button_copy">复制网址</a>
+<a class="jiathis_button_email">邮件</a>
+<a class="jiathis_button_weixin">微信</a>
+<a class="jiathis_button_qzone">QQ空间</a>
+<a class="jiathis_button_tqq">腾讯微薄</a>
+<a class="jiathis_button_douban">豆瓣</a>
+<a class="jiathis_button_share">一键分享</a>
```
Comment (ivan-nginx) [inline layout/_partials/share/jiathis.swig]: Can u translate this all labels in language dir?

File: layout/_partials/share/jiathis.swig [hunk 2]
```diff
@@ -1,21 +1,25 @@
...
+
+<a href="http://www.jiathis.com/share?uid=2140465" class="jiathis jiathis_txt jiathis_separator jtico jtico_jiathis" target="_blank">更多</a>
```
Comment (ivan-nginx) [inline layout/_partials/share/jiathis.swig]: And this too.

**Comment by elkan1788** [comment]
Thanks for your **CR**. But I don't think we need do this. This third party plugin only useful in China may not other language people. Also other share target will use in EN language. So is you still hold to your point？

**Comment by elkan1788** [comment]
Upgrade the analytics script code position, because someone maybe use icons display for analytics flag. So the script not suit in head mark.

File: layout/_layout.swig [hunk 3]
```diff
@@ -44,7 +43,7 @@
     <footer id="footer" class="footer">
       <div class="footer-inner">
         {% include '_partials/footer.swig' %}
-        {% include '_third-party/analytics/busuanzi-counter.swig' %}
+	{% include '_third-party/analytics/index.swig' %}
```
Comment (ivan-nginx) [inline layout/_layout.swig]: `busuanzi-counter` is visible widget for footer. No need to do this.

File: layout/_third-party/analytics/index.swig [hunk 4]
```diff
@@ -1,3 +1,4 @@
+{% include 'busuanzi-counter.swig' %}
```
Comment (ivan-nginx) [inline layout/_third-party/analytics/index.swig]: `busuanzi-counter` is visible widget for footer. No need to do this.

**Comment by elkan1788** [comment]
@ivan-nginx Sorry, maybe there you can't  understand my explain about [56a8d62](https://github.com/iissnan/hexo-theme-next/pull/1796/commits/56a8d620e48c072aba86aacb4c1399389e39e38b).  There not only `busuanzi-counter` is visible widget, also other analytics script will support visible icons or lable, just like **`Baidu`** or **`CNZZ`**. You can visit my blog [http://www.xn--7qv19ae78e.cn](http://xn--7qv19ae78e.cn) and scroll to footer, and I think you will known what I say, thanks.

**Comment by ivan-nginx** [comment]
@elkan1788 awesome site! China have own cyrrillic domains too, this is good.
About pull, mb i don't understand something, but as is see, u modify `_third-party/analytics/index.swig` file. But this file include many analytics widgets, like google. And there are at most invinsible, without any widgets. Let's see:
```

{% include 'facebook-sdk.swig' %}
{% include 'vkontakte-api.swig' %}
{% include 'google-analytics.swig' %}
{% include 'baidu-analytics.swig' %}
{% include 'tencent-analytics.swig' %}
{% include 'tencent-mta.swig' %}
{% include 'cnzz-analytics.swig' %}
{% include 'application-insights.swig' %}
```

1. FB is integrated as API. Like and comments post is supported.
2. VK same as FB.
3. GA does not have any widgets.
4. Baidu same as GA, i think.
5. Tencent same as Baidu, i think. 
6. CNZZ have visible widget, ok.
7. And AI same as GA, i think.

So, most part of this has no visible widget and if u want to see your CNZZ counter in footer, u must do it as separated option, like FB or VK API.

**Comment by elkan1788** [comment]
@ivan-nginx Thanks for suggest and do all update under your point. Please check. :)

File: layout/_third-party/analytics/analytics-with-widget.swig [hunk 5]
```diff
@@ -0,0 +1,4 @@
+{% include 'busuanzi-counter.swig' %}
+{% include 'busuanzi-counter.swig' %}
```
Comment (ivan-nginx) [inline layout/_third-party/analytics/analytics-with-widget.swig]: Why repeat `busuanzi-counter.swig`?

File: layout/_layout.swig [hunk 6]
```diff
@@ -44,7 +44,7 @@
     <footer id="footer" class="footer">
       <div class="footer-inner">
         {% include '_partials/footer.swig' %}
-        {% include '_third-party/analytics/busuanzi-counter.swig' %}
+        {% include '_third-party/analytics/analytics-width-widget.swig' %}
```
Comment (ivan-nginx) [inline layout/_layout.swig]: Maybe simple `analytics-widget.swig`?

File: layout/_third-party/analytics/index.swig [hunk 7]
```diff
@@ -1,3 +1,4 @@
...
 {% include 'facebook-sdk.swig' %}
 {% include 'vkontakte-api.swig' %}
 {% include 'google-analytics.swig' %}
```
Comment (ivan-nginx) [inline layout/_third-party/analytics/index.swig]: Also, if u move `tencent`, `cnzz` and other from index to widget, u must delete it from index, right?

File: layout/_third-party/analytics/analytics-with-widget.swig [hunk 8]
```diff
@@ -0,0 +1,4 @@
...
+{% include 'tencent-mta.swig' %}
```
Comment (ivan-nginx) [inline layout/_third-party/analytics/analytics-with-widget.swig]: Are u tested `tencent-mta.swig`? I'm not sure, what is visible widget. Please, test it to be sure and also need to test `tencent.swig`.

File: layout/_third-party/analytics/index.swig (hunk 4 above)
Comment (ivan-nginx) [inline layout/_third-party/analytics/index.swig]: Also, if u move `busuanzi-counter` index to widget, u must delete it from index, right?

**Comment by ivan-nginx** [comment]
@elkan1788 new comments added. PLease, test it all (especially both `tencent-mta`) to be sure all work's ok before adding/changing pull request.

**Comment by elkan1788** [comment]
@ivan-nginx  So sorry about that. Now fixed the bugs and trim all analytics scripts: make sure the Tencent analytics script support widget find out in office website.

**Comment by ivan-nginx** [comment]
@elkan1788 seems good. Good job and thanks!

**Pull request merged**